sonar-plugin-api/src/main/java/org/sonar/api/issue/Issuable.java

```diff
@@ -68,17 +67,39 @@
 
     /**
      * Optional line index, starting from 1. It must not be zero or negative.
+     * @deprecated since 5.2 use {@link #addLocation(NewIssueLocation)}
      */
+    @Deprecated
     IssueBuilder line(@Nullable Integer line);
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)